pkgs/top-level/emscripten-packages.nix

```diff
@@ -81,7 +81,9 @@ rec {
     });            
   
   xmlmirror = pkgs.buildEmscriptenPackage rec {
-    name = "xmlmirror";
+    name = "${pname}-${version}";
```

Comment:
`We` should just use pname in the expression below and drop this, if `buildEmscriptenPackage` allows.

Reply:
It doesn't. https://github.com/NixOS/nixpkgs/issues/72907

Reply:
That is just the `name` variables in the expression below. `buildEmscriptenPackage` does support `pname` + `version`:

https://github.com/NixOS/nixpkgs/blob/4835c83e3a1a855015fdeb2b236b03d70c490110/pkgs/development/em-modules/generic/default.nix#L13-L14

Reply:
Ah, i see. Thanks!